### PR TITLE
imgAlt filter

### PR DESCRIPTION
**Pull request by nicolas-san**
Little filter to fill alt attribute of img tag using keywords present in the img name.

(I PR also @fabpot fabpot/Twig-extensions, but I see after this repo as more recent modifications)

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sstok) [inline .idea/misc.xml]: This file adding should be reverted, add `/.idea` to your .git/info/exclude using.

`echo "/.idea" >> .git/info/exclude` in your command shell.

**Comment by sstok** [comment]
For the record `fabpot/Twig-extensions` is a fork of this repository ;)

**Comment by nicolas-san** [comment]
@sstok I see after PR that was a fork ^^

I delete .idea, thanks for pointing out !

**Comment by fabpot** [comment]
As the alt attribute is nowadays mostly used by screen readers, using a generated text seems like a bad idea. I'm :-1: on adding this.

**Comment by nicolas-san** [comment]
I understand your comment, but for now, I use this to fill the alt before I've nicer way to do it. I don't know if it's better than nothing in case of screen reader, I've no experience using them.

**Comment by stof** [comment]
-1 on this. The implementation is very specific to a special naming scheme for images. So it does not belong to a generic repository IMO

**Pull request closed without merging**